app/controllers/evaluations_controller.rb

```diff
@@ -32,68 +32,68 @@ def process_feedback
   def export_overall_course_rating
     if params[:provider].present? && params[:course_id].present?
       mooc_provider = MoocProvider.find_by!(name: params[:provider])
-      course = [Course.find_by!(provider_course_id: params[:course_id], mooc_provider: mooc_provider)].first()
+      course = [Course.find_by!(provider_course_id: params[:course_id], mooc_provider: mooc_provider)].first
```

Comment:
I agree to @rosabraatz' comment as find_by! always limits the SQL query to return one result. Why should we take one result, put it in an array (wich then will always have a size of zero or one) and get the first element (resulting in `nil` or the result from find_by!). Do you, @Atyanar, agree if I change the code here and in the method below?